app/models/total_donations.rb

```diff
@@ -0,0 +1,5 @@
+module TotalDonations
```

Comment:
It's awesome that you've found mixins, and how we can use them to DRY up our code. I think it is a bit early to make a judgement about whether this abstraction is useful or not, so I would hold off on adding it. In general, I am much more comfortable keeping some duplicated code in there than I am adding abstractions too early. 😀